### PR TITLE
Implement a dynamic email subject

### PR DESCRIPTION
**Pull request by williamdes**

### CODE REVIEW

**Comment by williamdes** [comment]
Should I also do this for the email title ?

I works fine, I got the emails using: `MF_EMAIL_SUBJECT="Nouvelles actualités ({new_items_count}) pour {feed_title}"`

**Comment by anson-vandoren** [comment]
Up to you if you think it's useful for the title as well.

Can you uncomment the `.env.sample` lines that are actually required to run correctly, though? Fine to leave the new var you added commented still

**Comment by williamdes** [comment]
> Up to you if you think it's useful for the title as well.

Yeah, you can translate it and do cool templating with this PR

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ feeds and have them delivered to their email inbox.
 
 ### Set up environment variables
 
-- `cp ./mailfeed/.env.sample ./mailfeed/.env`
+- `cp ./mailfeed/.env.dist ./mailfeed/.env`
```
Comment (williamdes) [inline README.md:17]: this was wrong anyway since the file was called: `mailfeed/.sample.env`

**Review by anson-vandoren — APPROVED** [review]

**Comment by anson-vandoren** [comment]
Thanks!!

**Pull request merged**